.github/workflows/iroha2-dev-pr.yml

```diff
@@ -43,7 +43,9 @@ jobs:
       - uses: actions/checkout@v4
       - uses: Swatinem/rust-cache@v2
       - name: Run tests, with coverage
-        run: mold --run cargo test --all-features --workspace --no-fail-fast
+        run: |
+          mold --run cargo clean
```

Comment:
why are you cleaning?

Reply:
if this is about #4488 . Let's please do it in separate PR

Reply:
I know, I'm just taking advantage of the opportunity to run CI